config/initializers/route_validators.rb

```diff
@@ -16,8 +16,6 @@ def self.matches?(request)
   end
 
   class VersionMatcher
-    include VersioningHelpers
```

Comment:
resolves DEPRECATION WARNING: Initialization autoloaded the constant VersioningHelpers.